### DISCUSSION TITLE
Is there any complete and articulated doc of Uno?

### DISCUSSION THREAD

**Discussion by juanyacovino77** [unoplatform/uno#12591] [General]
Hi there,

I am starter for Uno Platform, WinUI, MVVM pattern implementation, CommunityMvvmToolKit and XAML.

I thought that I could learn how to fully used Uno Platform framework from reading the official docs at www.platform.uno website (like it can be done with MAUI), but I could not found a COMPLETE doc or tutorial, I only found a bunch of external disordered links for reading but not a official doc that unifies and articulates all the concepts for developing Uno Platform apps at once... 

Is there anywhere what I am asking for? 

Thanks a lot

**Comment by jeromelaban** [comment]
Thanks for the request. Did you give a run to our getting started tutorials: https://platform.uno/docs/articles/getting-started-tutorial-1.html?

If you did, what would you like to see more of?

**Comment by juanyacovino77** [comment]
Well, you tell me. That shows nothing. 

First of all you would expect a suggestion on a MVVM pattern implementation with UNO maybe adding MVVM Community toolkit or Prism or smth. Then increasingly showing how to use more advanced features, like Commands, Navigation, Messengers but all these concept articulated under UNO framework.
